GliaWidgets/ViewController/Survey/SurveyViewController.swift

```diff
@@ -110,15 +117,15 @@ extension Survey {
 }
 
 extension Survey.QuestionPropsProtocol {
```

Comment:
I think it would be safer to use enum instead of protocol here. But I suppose it can be addressed later at some point.